_observing-your-data/query-insights/query-metrics.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+title: Query metrics
+parent: Query insights
+nav_order: 20
+---
+
+# Query metrics
+
+Query metrics involves comprehensive instrumentation along the search path to capture key [metrics](#metrics), such as aggregation types, query types, latency, and resource usage per query type. Open Telemetry (oTel) is the instrumentation framework. The telemetry data can be consumed using oTel metrics [exporters]({{site.url}}{{site.baseurl}}/observing-your-data/trace/distributed-tracing/#exporters).
```

Comment:
The first phrase of the first sentence is too vague and doesn't entirely work. How/in what way does "query metrics" involve something? What comprehensive instrumentation (is this information necessary?)? I would just combine the first two sentences.

Reply:
@natebower "Instrumentations" refers to the tools and processes used to collect, measure, and report data about the performance and behavior of a system. How about "Query metrics _uses_ comprehensive instrumentation along the search path to capture key [metrics](#metrics), such as aggregation types, query types, latency, and resource usage per query type."

Reply:
If "query metrics" are a generic noun, then it should be "Query metrics _use_", but that still reads oddly because we have metrics capturing other metrics. As written, we seem to be treating "Query Metrics" as a proper noun syntactically.

Reply:
OK, let's ask @deshsidd: Could you please address this comment? Thanks.

Reply:
Agree with @natebower last comment. @kolchfa-aws you can go forward with this change and we can merge this.